src/components/HomepageBanner.js

```diff
@@ -0,0 +1,224 @@
+            @media screen and (max-width: 900px) {
+              width: 90%;
+              top: 37%;
+            }
+          `}
+        >
+          <h1
+            css={css`
+              color: white;
+              font-size: min(4vw, 4rem);
+              @media screen and (max-width: 1200px) {
+                font-size: min(3.5vw, 2.5rem);
+              }
+              @media screen and (max-width: 900px) {
+                font-size: 2rem;
+                font-weight: 600;
+                width: 70%;
+              }
+            `}
+          >
+            Welcome to <br />
+            New Relic docs!
+          </h1>
+          <p
+            css={css`
+              color: white;
+              font-size: min(1vw, 1rem);
+              @media screen and (max-width: 1200px) {
+                display: none;
+              }
+              @media screen and (max-width: 900px) {
+                display: inline;
+                font-size: 14px;
+              }
+            `}
+          >
+            We have tons of content to get you started, feel free to browse
+            around. Don't forget to read our{' '}
+            <Link
+              to="/docs/using-new-relic/"
+              css={css`
+                color: white;
+              `}
+            >
+              introductory statement
+            </Link>{' '}
+            for a wider view on our platform's capabilities.
+          </p>
+        </div>
+        <div
+          css={css`
+            position: absolute;
+            bottom: 15%;
+            left: 5%;
+            z-index: 2;
+            display: flex;
+            @media screen and (max-width: 1500px) {
+              bottom: 10%;
+            }
+            @media screen and (max-width: 1200px) {
+              bottom: 15%;
+            }
+            @media screen and (max-width: 1000px) {
+              flex-direction: column-reverse;
+              align-items: flex-start;
+              justify-content: space-between;
+              height: 30%;
+            }
+            @media screen and (max-width: 900px) {
+              bottom: 3rem;
+              height: 15%;
+            }
+          `}
+        >
+          <Button
+            as={Link}
+            to="https://newrelic.com/signup"
+            css={css`
+              height: 40px;
+              font-size: 1rem;
+              background: var(--color-brand-100);
+              color: var(--color-brand-600);
+              &:hover {
+                background: var(--color-brand-500);
+                border-color: var(--color-brand-500);
+              }
+              @media screen and (max-width: 1500px) {
+                height: 30px;
+                font-size: 0.85rem;
+              }
+              @media screen and (max-width: 1100px) {
+                height: 25px;
+                font-size: 0.75rem;
+              }
+              @media screen and (max-width: 900px) {
+                height: 40px;
```

Comment:
We might look at making the buttons the same width < 900px width, but I'll def leave that up to @legnaleama

Reply:
Just double-checked the designs, disregard this!